platform/posix/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_library( clock_posix
                "clock_posix.c" )
 
 target_include_directories( clock_posix
-                              PRIVATE
```

Comment:
Why does this now need to be public?

Reply:
The demo uses headers from it; it should have been public in the first place as well